src/components/AssetHeader/AssetHeader.tsx

```diff
@@ -96,6 +96,7 @@ export const AssetHeader: React.FC<AssetHeaderProps> = ({ assetId, accountId })
                 {name} {`(${symbol}${asset.id ? ` ${middleEllipsis(asset.id)}` : ''})`}
               </Heading>
 
+              <WatchAssetButton assetId={assetId} />
```

Comment:
@reallybeard FYI, this adds watchlist capabilities on asset page for consistency with mobile

<img width="347" alt="Screenshot 2024-09-24 at 12 53 22" src="https://github.com/user-attachments/assets/af507d68-428b-4637-8fa2-dcee0d914be9">